QubitServer/src/main/java/org/labrad/qubits/QubitContext.java

```diff
@@ -21,7 +21,7 @@
 import org.labrad.qubits.channeldata.*;
 import org.labrad.qubits.channels.*;
 import org.labrad.qubits.config.MicrowaveSourceConfig;
-import org.labrad.qubits.config.MicrowaveSourceOffConfig;
+import org.labrad.qubits.config.MicrowaveSourceOffConfig$;
```

Comment:
I gather this dollar sign business is how Java deals with Scala singletons. It seems a bit ugly to me but I guess it's temporary.